tests/test_lib_subscriptions.py

```diff
@@ -103,9 +103,14 @@ def test_03_check_subscription(self):
 
         init_token({"type": "spass"}, user=User("shadow", self.realm1))
         init_token({"type": "spass"}, user=User("nopw", self.realm1))
-        # Now we have three users with tokens, subscription will fail
-        self.assertRaises(SubscriptionError, check_subscription,
-                          "demo_application")
+        # Now we have three users with tokens, but only two are allowed. We fail with a probabiliy of 1/3
```

Comment:
I think this is bound to produce randomly failing tests. We should mock out the randomness for the test and just check if we get the required error.

Reply:
done in ae1b513d